05-Time-Series-Raster.Rmd

```diff
@@ -202,64 +220,74 @@ xres(NDVI_stack)
 
 ```
 
-> ##Challenge
-> 
-> Before you go any further, answer the following questions about our `RasterStack`.
-> 
-> 1. What is the `CRS`?
-> 2. What is the `resolution` of the data? And what `units` is that resolution in?
+#Challenge: Raster Metadata
+Before going further, answer the following questions about our `RasterStack`.
 
-#Plotting Time Series Data
+1. What is the `CRS`?
+2. What is the `resolution` of the data? And what `units` is that resolution in?
+
+``` {r challenge-code-raster-metadata, eval=FALSE, echo=FALSE}
+#1. UTM zone 19 WGS 84
+#2. 30x30 meters
 
-Once we have created our `RasterStack`, we can visualize our data. Remember 
-from a previous lesson, that we can use the `plot` command to quickly plot a `RasterStack`.
+```
 
+#Plotting Time Series Data
+Once we have created our `RasterStack`, we can visualize our data. We can us
+the `plot()` command to quickly plot a `RasterStack`.
 
 ```{r plot-time-series }
 
-#view a histogram of all of the rasters
-#nc specifies number of columns
+#view a plot of all of the rasters
+#'nc' specifies number of columns (we will have 13 plots)
 plot(NDVI_stack, 
      zlim = c(1500, 10000), 
      nc = 4)
 
 ```
 
+Wait, NDVI data scale from 0-1, why is this data up to 10,000?  The metadata for this NDVI data have a scale factor: 10,000. A scale factor is
+commonly used in larger datasets to remove decimals. Storing decimal places
+actually consumes more space and creates larger file sizes compared to storing
+integer values.
 
-Note: we can make the `levelplot` even prettier by fixing the individual tile
-names. We will cover this in 
-[Lesson 06 - Plot Time Series Rasters in R ]({{ site.baseurl }}/R/Plot-Raster-Times-Series-Data-In-R/)
-{ : .notice }
+Note: We could make this plot (`levelplot`) even prettier by fixing the
+individual tile names and adding an overall plot title. How to do this is
+covered in the NEON Data Skills 
+[Plot Time Series Rasters in R ]({{ site.baseurl }}/R/Plot-Raster-Times-Series-Data-In-R/) 
+lesson. {: .notice2}
 
 ##Taking a Closer Look at Our Data
-
 Let's take a close look at the plots of our data. Given what you might know 
-about the seasons in Massachusettes (where Harvard Forest is located), do you 
+about the seasons in Massachusettes where the Harvard Forest is located, do you 
 notice anything that seems unusual about the patterns of greening and browning 
-of the vegetation atthe site?
+of the vegetation at the site?  
+
+Hint: the  number after the X in the tile title 
+is the Julian day.  If you are unfamiliar with Julian day, check out the NEON
+Data Skills 
+[Converting to Julian Day ]({{ site.baseurl }}/R/julian-day-conversion/) 
```

Comment:
Is this a lesson already built megan? I like the idea of having such a lesson. We don't have one on NDS however.